### DISCUSSION TITLE
Can't disable Virtualization-Based Security with Hyper-V installed?

### DISCUSSION THREAD

**Discussion by expend20** [HyperDbg/HyperDbg#178] [Q&A]
Hello, I tried everything from [this list](https://beebom.com/how-disable-virtualization-based-security-vbs-windows-11/) except uninstalling hyper-v and still Virtualization-Based Security is "running" for me.  I'm not running HyperDdb in a Hyper-V VM, I'm talking about the host PC.
I wonder can Hyper-V and HyperDbg work together? Or can Virtualization-Based Security feature be disabled without uninstalling Hyper-V?

Thanks

**Comment by SinaKarvandi** [comment]
Hi,
Thanks for reporting this.
Have you tried the steps mentioned here? https://docs.hyperdbg.org/getting-started/build-and-install#running-hyperdbg

> I wonder can Hyper-V and HyperDbg work together? Or can Virtualization-Based Security feature be disabled without uninstalling Hyper-V?

Unfortunately, no. We tried several times to add support of Hyper-V to HyperDbg but even though there was some progress, still not completely in a working state. So, you'll likely see BSOD after some time if you used HyperDbg in Hyper-v.

Please check the above link and let me know if you still have the problem so we can find a solution to solve it.

**Comment by expend20** [comment]
Hey,
Yeah, I've read the docs, but the documentation states that HyperDbg can't run **inside** of Hyper-V machine, but it doesn't state that Hyper-V can't be active on the host. But probably it's the same thing, I just kind of misunderstand how it really works.
Thanks for reply!
